### ISSUE TITLE
How to evaluate mAP on training set every epoch (coco-style dataset)

### ISSUE THREAD

**Issue by b02202050** [open-mmlab/mmdetection#1609]
Hi, when using `--validate`, only validation set will be evaluated. I wonder if there is any method to also do evaluation on training set after each epoch. (so I can see the result at the log)
Thanks!

Dataset type: coco-style

**Comment by hellock** [comment]
You can just modify the [evaluation dataset](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn_r50_fpn_1x.py#L138) in the config.

**Comment by DimTrigkakis** [comment]
That is not exactly useful, we have to re-run an entire training run just to check the metric on different validation datasets
